.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -2,14 +2,15 @@ name: Bug Report
 description: File a bug report
 title: "[Bug]: "
 labels: ["bug", "triage"]
-assignees: @openshift/openshift-team-windows-containers 
+assignees:
+  - "@openshift/openshift-team-windows-containers"
 body:
   - type: markdown
     attributes:
       value: |
         Thanks for taking the time to fill out this bug report! Before filling out this form,
-        please review the (Cluster and OS Pre-Requisites)[https://github.com/openshift/windows-machine-config-operator/blob/master/docs/wmco-prerequisites.md],
-        as well as the (BYOH Instance Pre-requisites)[BYOH Instance Pre-requisites].
+        please review the [Cluster and OS Pre-Requisites](https://github.com/openshift/windows-machine-config-operator/blob/master/docs/wmco-prerequisites.md),
+        as well as the [BYOH Instance Pre-requisites](BYOH Instance Pre-requisites).
```

Comment:
Now that I am able to see the rendered yaml, I suggest replacing the `Community issue` label with something like `Attention` or `Heads up`.  WDYT?